### DISCUSSION TITLE
Compile Tasmota - default SSID (WPA2/3)

### DISCUSSION THREAD

**Discussion by mattaipm** [arendst/Tasmota#20453] [Support request]
Where/how to change the configuration so that the Tasmota has the default SSID and configuration?
I did it from the manual, but it doesn't work.

**Comment by barbudor** [comment]
Sorry but your question is not clear
Which part of the manual are you refering to ?
Are you asking how to compile a binary with SSID hard-coded ?

If yes, you need to compile tasmota with the following lines in `user_config_override.h`:
```
#undef  STA_SSID1
#define STA_SSID1         "YourSSID"
#undef  STA_PASS1
#define STA_PASS1         "YourWifiPassword"
```

If this is not your question, please elaborate

**Reply by sfromis** [reply]
In the same way as SSID/Password, you can also set a number of other WiFi-related defaults at build time, like IP addresses. See
https://github.com/arendst/Tasmota/blob/development/tasmota/my_user_config.h#L67-L86
